repo/packages/B/beta-mke/0/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
```

Comment:
🤔 Shouldn't this be `5.0` like in https://github.com/mesosphere/universe/pull/1980?

Reply:
It can be 4.0 too, we shouldn't worry about it

Reply:
5.0 is only required for packages that requires a package manager. This is a package manager so doesn't require a package manager and do can stay at 4.0.